### PR TITLE
dependency-review

### PR DESCRIPTION
**Pull request by wallentx**

### CODE REVIEW

File: .github/workflows/dependency-review.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  contents: read
+
+jobs:
+  dependency-review:
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Checkout Repository'
+        uses: actions/checkout@v3
+      - name: 'Dependency Review'
+        uses: actions/dependency-review-action@v2
```
Comment (Starttoaster) [inline .github/workflows/dependency-review.yml:20]: I think this is already being performed by Snyk? https://docs.snyk.io/integrations/git-repository-scm-integrations/snyk-checks-on-pull-requests

Comment (wallentx) [reply .github/workflows/dependency-review.yml:20]:
```
 😎   Wait til you see the other 3 dependency review scanning tools I turn on
 ≤))≥
 _|\_
```

**Review by Starttoaster — APPROVED** [review]
LGTM

**Review by paninaro — APPROVED** [review]
lgtm

**Pull request merged**